Dockerfile

```
@@ -1,3 +1,3 @@
 FROM python:3.7-buster
-RUN pip install cairo-lang
+RUN pip install --upgrade cairo-lang
```

I don't think you need `--upgrade` here since a fresh install should pull the latest.

```suggestion
RUN pip install cairo-lang
```